pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -41,6 +43,22 @@ type SchedulerExtender interface {
 
 	// IsBinder returns whether this extender is configured for the Bind method.
 	IsBinder() bool
+
+	// ProcessPreemption returns nodes with their victim pods processed by extender based on
+	// given:
+	//   1. Pod to schedule
+	//   2. Candidate nodes and victim pods (nodeToVictims) generated by previous scheduling process.
+	// The possible changes made by extender may include:
+	//   1. Subset of given candidate nodes after preemption phase of extender.
+	//   2. A different set of victim pod for every given candidate node after preemption phase of extender.
+	// If you do not want to implement preemption, just return nodeToVictims.
```

Comment:
I think the last line is not needed. An extender that doesn't want to implement preemption will leave out the "preempt" verb in the configuration.

Reply:
Yep, removed.